### PR TITLE
Ubuntu and vagrant

### PR DESCRIPTION
**Pull request by pigmej**

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -44,38 +45,26 @@ MASTER_CPUS = cfg["master_cpus"]
...
-end
 
 def shell_script(filename, env=[], args=[])
   "/bin/bash -c \"#{env.join ' '} #{filename} #{args.join ' '} \""
 end
 
-solar_script = ansible_playbook_command("solar.yaml")
-# NOTE(bogdando) w/a for a centos7 issue
-fix_six = shell_script("/vagrant/deploy/solar/bootstrap/playbooks/fix_centos7_six.sh")
+# solar_script = ansible_playbook_command("solar.yaml")
```
Comment (nebril) [inline Vagrantfile]: You deleted ansible_playbook_command function, maybe just delete this line instead of commenting out?

File: Vagrantfile [hunk 2]
```diff
@@ -44,38 +45,26 @@ MASTER_CPUS = cfg["master_cpus"]
...
 
-master_pxe = ansible_playbook_command("pxe.yaml")
-get_solar = shell_script("/vagrant/deploy/get-solar.sh")
+# master_pxe = ansible_playbook_command("pxe.yaml")
```
Comment (nebril) [inline Vagrantfile]: Are these necessary for something? Maybe delete them?

File: resources/os_node/1.0.0/actions/run.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
+os=`lsb_release -si`
+
+if [[ ! "$os" == "Ubuntu" ]]
+then
+   sudo dnf install -y python python-dnf ansible libselinux-python
+   sudo /usr/sbin/setenforce 0
+else if [[ "$os" == "Ubuntu" ]]
+     then
```
Comment (nebril) [inline resources/os_node/1.0.0/actions/run.sh:9]: Is this proper indentation? 〰️

File: vagrant-settings.yaml_defaults [hunk 4]
```diff
@@ -1,14 +1,17 @@
...
+master_image: solar-project/solar-minimal
 master_image_version: null
-slaves_image: fc23
-slaves_image_version: null
-master_ram: 1024
+# 256 is enough, can be trimed down when needed
+master_ram: 350
 master_cpus: 1
+slaves_ram: 1024
+slaves_count: 1
```
Comment (nebril) [inline vagrant-settings.yaml_defaults]: Should be 2

File: Vagrantfile [hunk 5]
```diff
@@ -44,38 +45,26 @@ MASTER_CPUS = cfg["master_cpus"]
...
+# get_solar = shell_script("/vagrant/deploy/get-solar.sh")
```
Comment (nebril) [inline Vagrantfile]: ditto

File: resources/os_node/1.0.0/actions/run.sh (hunk 3 above)
Comment (pigmej) [inline resources/os_node/1.0.0/actions/run.sh:9]: It is :)

File: Vagrantfile (hunk 2 above)
Comment (pigmej) [inline Vagrantfile]: Done

File: Vagrantfile (hunk 1 above)
Comment (pigmej) [inline Vagrantfile]: Done

File: Vagrantfile (hunk 5 above)
Comment (pigmej) [inline Vagrantfile]: Done

**Pull request merged**